camlibs/Makefile.am

```diff
@@ -68,6 +68,9 @@ camlib_ldflags += -avoid-version
 camlib_ldflags += -export-dynamic
 camlib_ldflags += -export-symbols '$(top_srcdir)/camlibs/camlib.sym'
 camlib_ldflags += -rpath '$(camlibdir)'
+if HAVE_MMAN
```

Comment:
On mingw it needs linking with the libmman for the mmap/munmap which are needed by the st2205 camlib driver. This is simple workaround how to do it.